lib/samples/animate_images_with_image_overlay/README.md

```diff
@@ -0,0 +1,38 @@
+# Animate images with image overlay
+
+Animate a series of images with an image overlay.
+
+![Image of animate images with image overlay](animate_images_with_image_overlay.png)
+
+## Use case
+
+An image overlay is useful for displaying fast and dynamic images; for example, rendering real-time sensor data captured from a drone. Each frame from the drone becomes a static image which is updated on the fly as the data is made available.
+
+## How to use the sample
+
+The application loads a map of the Southwestern United States. Click the "Start" or "Stop" buttons to start or stop the radar animation. Use the drop down menu to select how quickly the animation plays. Move the slider to change the opacity of the image overlay.
+
+## How it works
+
+1. Create an `ImageOverlay` and add it to the `ArcGISSceneViewController`.
+2. Set up a timer with an initial interval time of 68ms, which will display approximately 15 `ImageFrame`s per second.
+3. When `Start` is clicked, start the timer.
+4. The timer updates the `ImageFrame` of the `ImageOverlay` at the given time interval.
+
+## Relevant API
+
+* ImageFrame
+* ImageOverlay
+* ArcGISSceneViewController
```

Comment:
This needs to be alphabetical (and then the metadata regenerated). I think that will resolve the PR checks